layers/meta-resin-ts/conf/samples/bblayers.conf.sample

```diff
@@ -7,7 +7,7 @@ BBFILES ?= ""
 
 BBLAYERS ?= " \
     ${TOPDIR}/../layers/poky/meta \
-    ${TOPDIR}/../layers/poky/meta-yocto \
```

Comment:
Looks like they did. It didn't error out though.
Re-commited now with using POKY_BBLAYERS_CONF_VERSION = "2" @agherzan 
Please re-review